src/components/forwardEvents.js

```diff
@@ -0,0 +1,70 @@
+// based on https://github.com/hperrin/svelte-material-ui/blob/master/packages/common/forwardEvents.js
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { bubble, listen } from "svelte/internal";
+
+export function forwardEvents(component, additionalEvents = []) {
+  const events = [
```

Comment:
Yikes :) Do you want a list of every event or just this particular list? There's a way to get the former but it's a little gnarly so this is fine.

Reply:
Well I guess it's harder if you only want input-related events, I don't see a great alternative except https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/oninput but that's only for `input events on the <input>, <select>, and <textarea> elements. It also handles these events on elements where contenteditable or designMode are turned on.`

Reply:
This is pretty standard and has not in practice caused any kind of bloat that I can tell. The Svelte Material UI library uses a similar thing. Open to thinking of another approach.